Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Suramar Flavor Pack + Impressions Diplomacy

## Changelog:
### General Suramar Flavor
- Shal'danalism (Nightborne faith) now allows concubines
- Added Elisande's clothing
- Gave Elisande Unique DNA
- Change Elisande's traits
- Adds Suramar to the isolation mechanic
   - If the Queen/King of Suramar is ruled by AI, they will automatically open the barrier in 615, as in lore, this is when Gul'dan contacted Suramar (no events for this yet though)
- Expanded the Isolation mechanic to include traveling for activities
- Expanded the Isolation mechanic to include wars
   - But if you have any existing wars, they should be able to be ended normally now
- Added **15+** unique events for the Ruler of Suramar and it's vassals
- Added a new decision for the ruler of Suramar: Remove the Arcane Barrier
   - To get access to this decision, the ruler of Suramar must learn of the outside world via the unique events
   - Once AI learns the decision, they _usually_ won't take it until after 615
- Add a Suramar character view
- Add several Suramar event backgrounds
- Adapted Court Brewmaster and allow anyone with Shal'danalism (Nightborne faith) to have breweries
- Added a new trait: Wretched
   - If you Banish characters as a ruler in Suramar, via the events, or imprisonment, they have a chance to become Wretched (if the arcandor tree is not restored)
   - If a nightborne travels outside of suramar they have the chance to become wretched as well
   - If the arcandor tree is not restored, random nightborne outside of suramar will have the chance to become wretched
   - This is to represent the nightfallen, and in the future, represent elves who are magic-starved (like the High Elves at some point)
- Added a new artifact: Seed of Arcandor
   - Get it from various events
   - Will get a boon from restoring the tree if you have it
- Add a decision: Restore the Arcandor Tree
   - Added special Arcandor Tree building

### New Mechanic: First Contact
- Expand the diplomatic range
  - EG: If you're in quel'thalas, you should be able to interact with most of Northrend and some of Pandaria
- Inspired by Stellaris' 'First Contact' event, certain people will be able to do 'first contact' diplomacy with certain rulers
- Some situations where First Contact is available and who can interact, right now, only in Suramar, but will extend to other situations and events
   - **De-Isolation of Suramar:** Any broken isles ruler (besides Suramar), (Duke or above, or player ruler) who is of elven culture or has an elven trait, in diplomatic range of suramar will be able to have contact with the ruler of Suramar upon it's de-isolation
- You can be friendly, hostile, or 'neutral' towards them on first contact
- You can contact them via letter event

# How to test:
- Test all of the new decisions and associated event chains.
- Test the First Contact Mechanic as the ruler + one who might receive the message, for all scenarios if possible
- The following standalone events will need testing. You can test them if you meet the requirements stated, by going into the console and using `event eventid` The Event ID is listed below
- `wc_suramar_events.0001` - Opening event - Requirement: Queen/King of Suramar
- `wc_suramar_events.0002` - Event to learn about the outside world - Requirement: Ruler in Suramar (Test as vassal AND queen/king)
- `wc_suramar_events.0003` - Event to learn about the nightfallen - Requirement: Ruler in Suramar (Test as vassal AND queen/king)
- `wc_suramar_events.0004` - Event where you become wretched (sometimes will happen if you travel outside of suramar) - Requirement: Ruler in Suramar (Test as vassal AND queen/king)
- `wc_suramar_events.0010` - Aqueduct/Vaults/Arcway event chain - Requirement: Ruler in Suramar (Test as vassal AND queen/king), several outcomes to test
- `wc_suramar_events.0020` - Thinking about Aszhara - Requirement: Ruler in Suramar (Test as vassal AND queen/king)
- `wc_suramar_events.0030` - Visited by an Elune Priestess - Requirement: Ruler in Suramar (Test as vassal AND queen/king), Could also test as kaldorei religion & Non-kaldorei religion for diff options
- `wc_suramar_events.0040` - Fal'dorei Event chain start Requirement: Ruler in Suramar (Test as vassal AND queen/king), not the holder of Meredil, Bonus: test with the callous trait
- `wc_suramar_events.0050` - *Rare Event* Vision of Sargeras - Requirement: Ruler in Suramar (Test as vassal AND queen/king), Could also test as sargerite & Non-sargerite for diff options
- `wc_suramar_events.0060` - Arcwine - Requirement: Ruler in Suramar (Test as vassal AND queen/king) - Test as a ruler who's already hosted a feast, and also test as a paranoid character
- `wc_suramar_events.0070` - Murloc event chain - Requirement: Ruler in Suramar, must not have arcane barrier desc unlocked yet, must not be a murloc, in a coastal county near a murloc coastal county
- `wc_suramar_events.0080` - A nightfallen seeks asylum in your court/camp - Requirement: Physically in Suramar (including landless adventurers)
- `wc_suramar_events.0100` - *Rare Event* Someone brings you a seed of arcandor - Requirement: Ruler in Suramar, must not have the seed of arcandor
- `wc_suramar_events.0110` - *Letter Event* A faldorei asks you to restore the tree - Requirement: King/Queen of Suramar, the barrier must be down and have neighboring faldorei, OR have a faldorei vassal
- `wc_suramar_events.0120` - You see a naga  - Requirement: Ruler in Suramar, must not be a naga, in a county near a naga county
- `wc_suramar_events.0130` - An adventurer in your realm - Requirement: Ruler in Suramar, there is a laamp encampment in your realm (non-nightborne leader)
